codelab/JinghangCodeLab/MagicNumberConverterTest.java

```diff
@@ -0,0 +1,29 @@
+package JinghangCodeLab;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class MagicNumberConverterTest {
```

Comment:
add a test case for empty string input, what should be the expected behavior for empty string? I recommend throw an IllegalArgumentException for this case. 

convert.binaryToDecimal("")